compiler+runtime/src/jank/clojure/core.jank

```diff
@@ -252,7 +252,9 @@
     ([]
      "")
     ([o]
-     (clojure.core-native/to-string o))
+     (if (nil? o)
+       ""
+       (clojure.core-native/to-string o)))
```

Comment:
We're already doing the `nil` check in the C++ code. The way I read it, this extra `nil?` check is not needed. Is that right?

Reply:
The trick here is that this line is calling `to-string`, which does not have the `nil` check. I added the `nil` check to `str`. So, I think this is right.